doc/howto_release.md

```diff
@@ -315,18 +320,18 @@ Software pages:
      copy_addon 820RC1 8.2.0RC1
 ```
 
-#### Ubuntu Launchpad notes
+### Ubuntu Launchpad notes
 
 - Create milestone and release: <https://launchpad.net/grass/+series>
 - Upload tarball for created release
 
-#### Packaging notes
+### Other notes
 
 - <https://trac.osgeo.org/grass/wiki/BuildHints>
     - <https://trac.osgeo.org/grass/wiki/DebianUbuntuPackaging>
     - <https://trac.osgeo.org/grass/wiki/CompileOnWindows>
 
-#### Marketing - tell others about release
+## Tell others about release
 
 - Notify all packagers (MN has email list)
```

Comment:
```suggestion
```
(no longer desired, they may follow GitHub or grass-dev/grass-accounce)

Reply:
Done in df482ad.